templates/.github/rename-to-workflows/readme.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 Google LLC
```

Comment:
This is copy from existing job, replacing yoshi-code-bot token with the default token.